### ISSUE TITLE
TJ +99200nnnnnnn numbers not recognized

### ISSUE THREAD

**Issue by impalex** [twcclegg/libphonenumber-csharp#110]
Library version: 8.11.2
Phone number example: +992000066362, mobile number, operator - MEGAFON
Google library output: https://libphonenumber.appspot.com/phonenumberparser?number=%2B992000066362 — no issue

However, libphonenumber-csharp results differs:
```csharp
            var parser = PhoneNumberUtil.GetInstance();
            var phone = parser.Parse(@"+992000066362", string.Empty);
            var code = parser.GetRegionCodeForNumber(phone);
            // code = 'TJ' — CORRECT
            var isPossible = parser.IsPossibleNumber(phone);
            // isPossible = true — CORRECT
            var isValid = parser.IsValidNumber(phone);
            // isValid = false — INVALID, should be true
            var e164num = parser.Format(phone, PhoneNumberFormat.E164);
            // e164num = '+992066362' — INVALID, should be '+992000066362'
```

**Comment by twcclegg** [comment]
I'm able to reproduce. I'll try to find some time to look into it.

**Comment by impalex** [comment]
I think I fixed it - #111

**Comment by twcclegg** [comment]
The fixed will be released in 8.11.3 whenever Google releases their version (probably about two weeks) in the meantime, if need be it can be fetched from https://ci.appveyor.com/project/twcclegg/libphonenumber-csharp/builds/30287711/artifacts
